src/mame/galaxian/galaxian.cpp

```diff
@@ -710,6 +710,52 @@ t) 'bagmanmc'
   - DSW bit 6 was previously used for "Bonus Lives" settings, but it has no effect
     in this set because of 'NOP' instructions from 0x3501 to 0x3507.
 
+u) uniwars
+
+    Uniwars Warp Analysis on an Irem PCB from PrSwan (Paul Swan)
+    ------------------------------------------------------------
+
+    - What is the warp effect?
+      The game PCB has several fly wires of which Red & Green relate to the warp effect.
+
+      Video PCB connections
+      Red   => IC K8 (7420)  pin 12
+      Green => IC F6 (74161) pin 14
+
+      The Green signal is a clock generated on the video PCB and the 161 is likely one of the video H/V counters.
+      It measured 10.44us.
```

Comment:
*What* measured 10.44 microseconds?  The pulse width?  The period?  Something else?  What’s the duty cycle?  What divider chain on the board could be producing this signal?  What master clock is it derived from?